_sass/_global-elements.scss

```diff
@@ -0,0 +1,20 @@
+// ==========================================================================
+// Styling for global page elements - Ex: Figure
+// ==========================================================================
+figure {
+
+  img {
+    display: block;
+  }
+
+  figcaption {
+    // USWDS utility to set family and font-size
+    // https://designsystem.digital.gov/design-tokens/typesetting/font-size/
+    @include u-font-size("ui", "xs");
+    // USWDS utility that adds 16px padding on sides
+    @include u-padding-x(2);
+    background-color: color("base-lightest");
+    padding-top: units(1);
+    padding-bottom: units(1.5);
+  }
```

Comment:
6. USWDS offers so many nice tools to help us theme custom elements and still have them _feel_ like part of the framework. Here I'm using some utilities to help me set the font-size, font family, and side padding for the figcaption. You can read more about [font utilities here →](https://designsystem.digital.gov/utilities/font-size-and-family/#utility-mixins) and [padding utilities →](https://designsystem.digital.gov/utilities/margin-and-padding/).

You'll also see I'm using design tokens for spacing. You can find a complete guide [here →](https://designsystem.digital.gov/design-tokens/spacing-units/)